src/coreclr/jit/optimizer.cpp

```diff
@@ -5439,8 +5547,9 @@ void Compiler::optCloneLoop(unsigned loopInd, LoopCloneContext* context)
     optPerformStaticOptimizations(loopInd, context DEBUGARG(true));
 
     // Now go through the new blocks, remapping their jump targets within the loop.
-    for (BasicBlock* blk = optLoopTable[loopInd].lpFirst; blk != optLoopTable[loopInd].lpBottom->bbNext;
-         blk             = blk->bbNext)
+    for (BasicBlock* blk = optLoopTable[loopInd].lpFirst; //
```

Comment:
remove the extra `//`  present at the end of this line and next line?

Reply:
This is a hack to keep jit-format from merging the three lines into 2. I'm ok with either 1 line or 3, but 2 for a `for` is annoying, IMO.